platforms/melpomene/src/main.rs

```diff
@@ -94,8 +94,11 @@ async fn kernel_entry() {
         tracing::warn!("Not spawning TCP UART server!");
     }
 
+    let mut debounce_period = Duration::from_millis(50);
+
     // Spawn the graphics driver
     if config.platform.display.enabled {
+        debounce_period = Duration::from_secs(1) / config.platform.display.frames_per_second as u32;
```

Comment:
it would be neat if there was eventually a way to do this generically by having the shell ask the display what its frame rate is...but we should save that for a future PR, probably.